src/browser_action/on-each-interaction.js

```diff
@@ -11,31 +11,71 @@
  limitations under the License.
 */
 
+import {INPThresholds} from './web-vitals.js';
+
 /**
  * @param {Function} callback
  */
 export function onEachInteraction(callback) {
-  const valueToRating = (score) => score <= 200 ? 'good' : score <= 500 ? 'needs-improvement' : 'poor';
+  const valueToRating = (score) => score <= INPThresholds[0] ? 'good' : score <= INPThresholds[1] ? 'needs-improvement' : 'poor';
 
   const observer = new PerformanceObserver((list) => {
```

Comment:
Nit: you might want to call this `eventObserver` now that you also have a LoAF observer.

Reply:
Done.